invenio_i18n/templates/invenio_i18n/macros/language_selector.html

```diff
@@ -0,0 +1,23 @@
+
+{% macro language_render() %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    {{ caller("javascript:document.getElementById('form_invenio_i18n_lang_{0}').submit();".format(language_item[0]), language_item[1]) }}
+  {% endfor %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    <form id="form_invenio_i18n_lang_{{ language_item[0] }}"
```

Comment:
I like the button option with a ``btn-link`` in the CSS class (or perhaps just add a ``css_class`` parameter to the macro which defaults to ``btn btn-link``.

Also, can it be configurable if it uses lang code or local display name inside the button: (``EN | DA``) vs. (``English, dansk``).